src/modules/PortAudio/PortAudioWriter.cpp

```diff
@@ -43,10 +46,21 @@ PortAudioWriter::PortAudioWriter(Module &module) :
 	outputParameters.sampleFormat = paFloat32;
 	outputParameters.hostApiSpecificStreamInfo = nullptr;
 
+#ifdef Q_OS_MACOS
+	coreAudioDevice = nullptr;
```

Comment:
This is already initialized in header file :)

Reply:
Well, you asked for it :p

On second thought, I'm moving the initialisation up so that all initialised member variables are handled in the same way in the same location. I don't really like initialising variables in a header; too easy to forget.

Reply:
> Well, you asked for it

I asked for initialization in header, because QMPlay2 uses now C++11 and we can use new features. Initializing POD data in constructor to its default values (0) makes no sense, so hopefully C++11 helps us allowing to initialize in header :)